core/src/nucleus/mod.rs

```diff
@@ -358,6 +360,18 @@ fn reduce_execute_zome_function(
     }
 }
 
+fn get_zome_name_for_entry_type(dna: &Dna, entry_type: String) -> Option<String> {
```

Comment:
I think this should actually be a method of `Dna`.
Would you mind moving this to `dna/src/lib.rs` ?

Reply:
I agree. Will do.

Reply:
Done, and also added a test.